src/app/runtime/wac/services/powershell-service.ts

```diff
@@ -119,9 +119,14 @@ export class PowershellService {
       }),
       catchError((e, _) => {
         let rethrow = e;
-        // WAC wrap the powershell error message around this AjaxError object. We would unwrap it for easier readability
-        if (e.name == "AjaxError" && e.status == 400 && e.response.error) {
-          rethrow = e.response.error;
+        // WAC wrap the powershell error (or exception) message around this AjaxError object. We would unwrap it for easier readability
+        if (e.name == "AjaxError" && e.status == 400) {
+          if (e.response.error && !e.response.exception) {
+            rethrow = e.response.error;
+          } 
+          if (!e.response.error && e.response.exception) {
+            rethrow = e.response.exception;
+          }
         }

```

Comment:
Looks like `error` is present if script fails with `Write-Error`. `exception` is present if script fails with `throw`.
We need to verify this assumption and always check if they both exists, output warning if that happens.
The script as is only silently override `rethrow` variable and swallow the problem

Reply:
Another solution would be what we talked about. If they are both present. Do not unwrap

Reply:
Fixed.